packages/database/src/migrations/20210816012839-AddProjectPacmossi-modifies-data.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+import { codeToId, insertObject, generateId, findSingleRecord, deleteObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const countryCodes = ['FJ', 'KI', 'NR', 'NU', 'PG', 'SB', 'TV', 'TO', 'VU'];
+
+const projectCode = 'pacmossi';
+const projectName = 'PacMOSSI';
+const projectDescription = 'Pacific Mosquito Surveillance Strengthening for Impact';
+
+const userGroups = ['PacMOSSI', 'PacMOSSI Senior'];
+
+const dashboardName = 'Vector Surveillance';
+const projectDashboardItemCode = 'project_details';
+const projectDashboardCode = `${projectCode}_${projectName}`;
+
+const getCountryDashboardCode = (countryCode, userGroup) =>
+  `${countryCode}_PacMOSSI_Country_${userGroup.split(' ').join('_')}`;
```

Comment:
We can now define user permissions `dashboard_relation` so would suggest removing user suffix. And `Country` was also relic of old system.  ``${countryCode}_${project}_${name}` seems a common pattern.
```suggestion
  `${countryCode}_PacMOSSI_${dashboardName.split(' ').join('_')}`;
```

Reply:
Have changed.